api/all.yaml

```diff
@@ -2082,6 +2140,151 @@ components:
       required:
         - full_name
         - version
+    CreateUser:
```

Comment:
I'm asking partly out of curiosity, but in relation to this [issue](https://github.com/unitycatalog/unitycatalog/pull/290) of mine, for example, is the `required` property not necessary for this definition?

Reply:
Nice catch... we should do the same thing here... I'll make the updates. Actually, they will probably be in this PR https://github.com/unitycatalog/unitycatalog/pull/387 which will eventually be reflected here.

Reply:
Sorry for leaving a comment out of the blue. I enjoy reading source code, so I was going through it. I just wanted to ask if there are any best practices or policies related to API design, particularly in relation to my PR. If I was interrupting your work, I apologize.

Reply:
No problem. Feel free to question/comment.